content/ko/docs/contribute/style/write-new-topic.md

```diff
@@ -104,9 +104,9 @@ YAML 블록이다. 여기 예시가 있다.
   파드를 만들기 위해 YAML 파일을
   포함할 수 있다.
 - 이 코드의 목적은 더 큰 파일의 일부를 강조하는 것이기 때문에
-  불완전한 예제다. 예를 들어 몇 가지 이유로
-  [PodSecurityPolicy](/ko/docs/tasks/administer-cluster/sysctl-cluster/#파드시큐리티폴리시-podsecuritypolicy)
-  를 사용자 정의 방법을 설명할 때 문서 파일에서 직접 짧은 요약 정보를 제공할 수 있다.
+  불완전한 예제다. 예를 들어
```

Comment:
[원문]
> The code is an incomplete example because its purpose is to highlight a portion of a larger file.

조금 더 자연스럽게 해 보았습니다. 근데 쉽지 않네요.. ㅎㅎ

"이 코드는 불완전한 예제인데, 이는 이 코드의 목적이 큰 파일의 일부분을 강조하는 것이기 때문이다."

Reply:
이 부분에 있는 항목들은, 마크다운으로 코드를 삽입하는 경우에 대한 예시들이기 때문에 모두 아래와 같이 변경했습니다!
- `이 코드는 ~~ 하다.`  ---> `- ~~한 코드.`